src/protocol.rs

```diff
@@ -252,7 +285,7 @@ impl Timestamp {
 ///     primitives::{PrimitiveStyle, Rectangle, Styled},
 /// };
 ///
-/// let mut buffer = PayloadBuffer::new();
+/// let mut buffer = PayloadBuffer::default();
```

Comment:
There is no longer a reason to switch to the `default` method. The existing code still works.